README.md

```diff
@@ -71,9 +71,9 @@ The topics will be inferred by the browser. The browser will leverage a classifi
     * The request header will not modify state for the caller unless there is a corresponding response header. That is, the topic of the page won't be considered observed, nor will it affect the user's topic calculation for the next epoch. 
     * The response header will only be honored if the corresponding request included the topics header (or would have included the header if it wasn't empty).
     * The registrable domain used for topic observation is that of the url of the request.
-    * Example request header: `Sec-Browsing-Topics: 123;v=chrome.1:1:2, 2`
+    * Example request header: `Sec-Browsing-Topics: t=(123;v=chrome.1:1:2 2;v=chrome.1:1:2), p=P000000000000000000000000`
```

Comment:
So, I realized that `:` is not allowed in structured field tokens. Probably you need to change them to be strings instead.

Reply:
I think the spec does allow ":" and "."? https://www.rfc-editor.org/rfc/rfc8941.html#section-3.3.4

Reply:
Sorry for missing this; you're right!